### ISSUE TITLE
chromium: No GSettings schemas are installed on the system

### ISSUE THREAD

**Issue by NeQuissimus** [NixOS/nixpkgs#26560]
## Issue description
I think this is similar to #25518

Whenever I try to access the system print dialog or want to upload a file, Chromium crashes on me with the below message.

```
(chromium:19213): GLib-GIO-ERROR **: No GSettings schemas are installed on the system [1] 19213 trace trap ./result/bin/chromium
```

I tried to fix this using the solution in #25518 but a Chromium build takes my poor little laptop > 12 hrs and my first two attempts were unsuccessful.

### Steps to reproduce
- Start Chromium
- Open some sort of file upload dialog

## Technical details

* System: 17.03.1313.d10fe64124
* Nix version: nix-env (Nix) 1.11.10
* Nixpkgs version: "17.03.1298.5bafc71b61" 
* Sandboxing enabled: build-use-sandbox = true

**Comment by 7c6f434c** [comment]
You don't need to rebuild Chromium per se, it should be enough to extend the wrapper with 
```
--prefix XDG_DATA_DIRS : "$GSETTINGS_SCHEMAS_PATH" \
--suffix XDG_DATA_DIRS : '${gnome3.defaultIconTheme}/share'
```

**Comment by cstrahan** [comment]
Fixed in #26614
